### PR TITLE
fix(#886) - Home-SPA Modal has incorrect link and add get-started to SSI

### LINKED ISSUE #886: Home-SPA Modal has incorrect link and add get-started to SSI
## Expected behavior

- Should point to the correct link i.e. `/get-started`

### PR DESCRIPTION
**Pull request by hybridx**
### Resolves #886 

# Explain the feature/fix

Changed ion-icons to images and fixed get-started link

## Does this PR introduce a breaking change

No

## Screenshots

Screenshot 2021-02-19 at 6 44 33 PM

### Ready-for-merge Checklist

- [x] Expected files: all files in this pull request are related to one feature request or issue (no stragglers)?
- [x] Does the change have appropriate unit tests?
- [x] Did tests pass?
- [x] Did you update or add any necessary documentation (README.md, WHY.md, etc.)?
- [x] Was this feature demo'd and the design review approved?

### CODE REVIEW

File: packages/ssi-service/src/nav/nav.css [hunk 1]
```diff
@@ -172,6 +172,7 @@
 .op-nav__icon {
   display: block;
   font-size: 1.25rem;
```
Comment (deshmukhmayur) [inline packages/ssi-service/src/nav/nav.css:174]: You can remove the font-size as we're using the svg icon instead of the font icon now.

**Review by deshmukhmayur — CHANGES_REQUESTED** [review]
Can you divide the commit into 2 different commits, 1 for the home-spa url fix, and the other for the Nav change.

File: packages/ssi-service/src/nav/nav.css (hunk 1 above)
Comment (hybridx) [reply packages/ssi-service/src/nav/nav.css]: @deshmukhmayur Correct. The changes have been made can you please review?

**Review by deshmukhmayur — APPROVED** [review]
LGTM.

File: packages/ssi-service/src/nav/nav.js [hunk 2]
```diff
@@ -317,13 +317,22 @@ window.customElements.define( 'op-nav', class extends LitElement {
 
           <nav class="op-menu">
             <ul class="op-menu-wrapper">
+              <li class="op-menu__item">
+                <a type="button"
+                  class="op-menu__item-button"
+                  target="_blank"
+                  href="/get-started">
+                  <img src="${rocketOutline}" class="op-nav__icon" alt="rocket-outline">
+                  <span>Get-Started</span>
```
Comment (ghanlohar) [inline packages/ssi-service/src/nav/nav.js:326]:
```suggestion
                  <span>Get Started</span>
```

**Review by ghanlohar — APPROVED** [review]

File: packages/ssi-service/src/nav/nav.js (hunk 2 above)
Comment (hybridx) [reply packages/ssi-service/src/nav/nav.js]: Thanks @ghanlohar

**Review by riginoommen — APPROVED** [review]

**Pull request merged**